sixtools/preliminary_processing.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+# This dictionary should eventually disappear, we should have this region
+# dictionary saved as a configuration attribute of rixscam. At this point
+# we should change the default for regions in extract_regions to None.
+# Also the manual is really confusing regarding what numbers need to go here,
+# we should discuss this.
+regions = {'dark_low2theta': np.s_[2:1635, 4:1601],
+           'data_low2theta': np.s_[1651:3286, 4:1601],
+           'dark_high2theta': np.s_[2:1635, 4:1601],
+           'data_high2theta': np.s_[1651:3286, 4:1601]}
+
+
+def extract_region(raw_data, region, regions=regions):
```

Comment:
It is very bad practice to have mutable defaults.  They are bound at function definition time and if you mutate them it persist 

```python
def surprise(inp=[0]):
    inp[0] += 1
    print(inp)
```

Reply:
The singular name vs plural behavior is also a bit surprising.